2020/13xxx/CVE-2020-13663.json

```diff
@@ -1,18 +1,79 @@
 {
```

Comment:
{} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': 'Cross Site Request Forgery'}]}]}